deploy/webhook.yaml

```diff
@@ -0,0 +1,265 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: service-apis
+  labels:
+    name: service-apis
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: service-apis-admission
+webhooks:
+  - name: validate.networking.x-k8s.io
+    matchPolicy: Equivalent
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: [ "networking.x-k8s.io" ]
+        apiVersions: [ "v1alpha1" ]
+        resources: [ "httproutes" ]
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: service-apis-admission
+        namespace: service-apis
+        path: "/validate"
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: service-apis-admission
+  annotations:
```

Comment:
Since there are not annotations, please remove this.

Reply:
Oops, good catch